### PR TITLE
docs: Format example for tracer.startActiveSpan

### PR DESCRIPTION
**Pull request by ad-m**
Example in documentation fro `tracer.startActiveSpan` is misformated:

image

That PR is gonna fix that:

image

I have doubts whether the second example - although technically correct - is optimal for promotion and indication in the documentation.

### CODE REVIEW

**Review by naseemkullah — APPROVED** [review]

**Comment by naseemkullah** [comment]
> I have doubts whether the second example - although technically correct - is optimal for promotion and indication in the documentation.

The idea was to highlight that you could return the span if you wanted to continue to manage it outside of the function, but improvements to the examples are always welcome!

**Review by vmarchaud — APPROVED** [review]

**Comment by Rauno56** [comment]
> > I have doubts whether the second example - although technically correct - is optimal for promotion and indication in the documentation.
> 
> The idea was to highlight that you could return the span if you wanted to continue to manage it outside of the function, but improvements to the examples are always welcome!

OP didn't mean a conceptual change. Only formatting!
Thanks, @ad-m!

Doesn't have to be in this PR, but I personally appreciate examples that run more than pseudo-code ones. Making minimal changes(console.log'ing instead of "do some work" and throwing etc.) could make this look a little better.

**Review by Rauno56 — APPROVED** [review]

**Comment by ad-m** [comment]
> OP didn't mean a conceptual change. Only formatting!

I perceive that PR as only formatting change. 

Despite of that I added side note about second example, but I believe it is out of scope of that PR. 

> The idea was to highlight that you could return the span if you wanted to continue to manage it outside of the function, but improvements to the examples are always welcome!

Thanks for the clarification. This may actually be useful on rare occasions.

**Comment by naseemkullah** [comment]
> > > I have doubts whether the second example - although technically correct - is optimal for promotion and indication in the documentation.
> > 
> > 
> > The idea was to highlight that you could return the span if you wanted to continue to manage it outside of the function, but improvements to the examples are always welcome!
> 
> OP didn't mean a conceptual change. Only formatting!

While their PR targets formatting they also inquired about the second example (conceptually) :) 

> Doesn't have to be in this PR, but I personally appreciate examples that run more than pseudo-code ones. Making minimal changes(console.log'ing instead of "do some work" and throwing etc.) could make this look a little better.

Yep, up for a PR?

**Review by johnbley — APPROVED** [review]

File: src/trace/tracer.ts [hunk 1]
```diff
@@ -50,36 +50,37 @@ export interface Tracer {
...
+   *       } catch (err) {
+   *         span.setStatus({
+   *           code: SpanStatusCode.ERROR,
+   *           message: err.message,
+   *         });
+   *         throw err;
+   *       }
+   *     });
+   *     do some more work
+   *     span.end();
```
Comment (dyladan) [inline src/trace/tracer.ts:83]: I would remove this example. When using startActiveSpan I expect the span to be ended most of the time within the callback.

Comment (dyladan) [reply src/trace/tracer.ts:83]: Nevermind I see you only want to do formatting here.

**Review by dyladan — APPROVED** [review]

**Comment by ad-m** [comment]
Thanks for your cooperation.

**Pull request merged**